examples/gallery/demos/bokeh/mandelbrot_section.ipynb

```diff
@@ -39,7 +39,7 @@
     "try:    from urllib2 import urlopen\n",
     "except: from urllib.request import urlopen\n",
     "\n",
-    "raw = urlopen('http://assets.holoviews.org/data/mandelbrot.npy').read()\n",
+    "raw = urlopen('https://assets.holoviews.org/data/mandelbrot.npy').read()\n",
```

Comment:
Did you test these? There's a few of them.

Reply:
No I haven't...I'll just switch those back to http as well...